index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo List</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href= "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="container">
+        <div class="app">
+            <h2>Todo List</h2>
+            <form>
+                <input type="text" placeholder="Enter your task">
+                <button type="submit">Add</button>
+            </form>
+            <ul>
+                <li><input type="checkbox">Item 1 <i class="fa fa-trash"></i></li>
+                <li><input type="checkbox">Item 1 <i class="fa fa-trash"></i> </li>
+                <li><input type="checkbox">Item 1 <i class="fa fa-trash"></i></li>
```

Comment:
Just having input type as checkbox will not be enough for this to work. more on that here: 
- https://www.w3schools.com/tags/att_input_type_checkbox.asp
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

Reply:
Added new features to the checkbox.